Frontend/raytrashing.html

```diff
@@ -137,23 +137,31 @@ <h4>Camera</h4>
 	</div>
 </div>
 
-<div id = "right">
-	<div>
-		<h4 style = "color: black; margin-left:20%; margin-right: 15%; float:left">Lights</h4>
-		<h4 style = "color: black; margin-left:20%; display:inline; float:left">Shapes</h4>
-		<select id="Lights" onclick="lightOption()" style = "display:inline; margin-top: 20%; margin-left:-70%; float:left; width:25%" size ="3"></select>
-		<select id="Shapes" onclick="shapeOption()" style = "display:inline; margin-top: 20%; margin-left:-20%; float:left; width:25%" size ="3"></select>
+<div id = "right" style="margin-left: 200px;">
+	<div style="margin-left: 10%;">
+		<h4 style = "color:black;">Shapes</h4>
+	
+		<select id="Shapes" style = "width:50%;" onclick="shapeOption()"  size ="3"></select>
+
+	
+		<h4 style = "color:black;">Lights</h4>
+		
+		<select id="Lights" style = "width:50%;" onclick="lightOption()" size ="3"></select>
+		
 	</div>
 
-	<div style="color: black; margin-top: 60%; margin-left:2%;">
+	<div style="color: black;margin-left:10%;">
 		<h4 style="color: black;">Coordinates</h4>
-		<input id="change_x" style = "width: 15%;" type="text" placeholder="x">
-		<input id="change_y" style = "width: 15%; margin-left:1%;" type="text" placeholder="y">
-		<input id="change_z" style = "width: 15%; margin-left:1%;" type="text" placeholder="z">
-		<input id="change_s" style = "width: 15%; margin-left:1%;" type="text" placeholder="s">
-		<br><br>
-		<button style = "text-align: center; width: 35%" class ="buttons" onclick="changeItem()">Change</button>
-		<button style = "text-align: center; width: 35%" class ="buttons" onclick="deleteItem()">Delete</button>
+		
+		<div style="color: black; margin-left:-3%; margin-top:-10px;">
+			<input id="change_x" style = "width:15%; height:15%; margin-left:-65%;" type="text" placeholder="x">
+			<input id="change_y" style = "width:15%; height:15%;" type="text" placeholder="y">

```

Comment:
Looks good. Just make sure X, Y, and Z are in the same line. Size can be in the next line. Also, make sure the placeholder for size is Size and not S. Thanks!